pyproject.toml

```diff
@@ -9,8 +9,12 @@ readme = "README.md"
 [tool.poetry.dependencies]
 python = "^3.7"
 opencv-python = "^4.7.0.68"
-rpi-ws281x = {version = "^4.3.4", platform = "linux"}
-rpi-ws281x-mock = {version = "^0.2.2", platform = "win32"}
+rpi-ws281x = { version = "^4.3.4", platform = "linux", markers = "platform_machine == 'arm64' or platform_machine == 'arm32'" }
+rpi-ws281x-mock = [
+  { version = "^0.2.2", platform = "win32" },
+  { version = "^0.2.2", platform = "darwin" },
+  { version = "^0.2.2", platform = "linux", markers = "platform_machine == 'amd64'" },
```

Comment:
The amd64 marker is giving me issues when I try to install on my Ubuntu 22.04 VM because neither rpi-ws281x or rpi-ws281x-mock never installs. Removing that part fixes installation on the VM.

Reply:
Got it, removing now, maybe that's just unnecessary if the previous dependency is checked first.

Reply:
I'll confirm this works on the matrix later tonight. Thank you!